wasm/wasi/wit/deps/hermes-sqlite/api.wit

```diff
@@ -0,0 +1,274 @@
+// cspell: words ioerr cantopen toobig nolfs notadb maindbname lookaside fkey qpsg scanstatus
```

Comment:
Can we move cpell words to project.dic, please?

Reply:
Basically you can validate those structs and functions interfaces from this docs: [https://www.sqlite.org/cintro.html](https://www.sqlite.org/cintro.html) with additional functions that are useful [sqlite3_db_config](https://www.sqlite.org/c3ref/db_config.html), and [sqlite3_db_status](https://www.sqlite.org/c3ref/db_status.html). Error status is [here](https://www.sqlite.org/rescode.html). The `open()` function doesn't require the filepath parameter as Hermes itself internally provides for this. other functions implement interfaces according to the C interfaces provided in the docs.